examples/pxScene2d/src/pxScene.cpp

```diff
@@ -198,7 +200,12 @@ class sceneWindow : public pxWindow, public pxIViewContainer
     if (gDumpMemUsage)
     {
       rtLogInfo("pxobjectcount is [%d]",pxObjectCount);
+
+#ifdef PX_PLATFORM_MAC
+      rtLogInfo("texture memory usage is [%lld]",context.currentTextureMemoryUsageInBytes());
```

Comment:
Let's change %lld to PRId64.  This is the desired define to use to print int64_t, and it will eliminate the need for a platform #ifdef here.